run_kraken.py

```diff
@@ -38,6 +40,17 @@ def main(cfg):
                                          "'s/\x1B\[([0-9]{1,3}(;[0-9]{1,2})?)?[mGK]//g'") # noqa
         logging.info("\n%s%s" % (cluster_version, cluster_info))
 
+        if install_and_enable_cerberus:
+            try:
+                runcommand.invoke("podman ps | grep -w cerberus || podman ps -a | grep -w cerberus")
+                runcommand.invoke("podman rm -f cerberus")
+            except:
+                pass
+            runcommand.invoke("podman pull quay.io/openshift-scale/cerberus")
+            runcommand.invoke("podman run --name=cerberus --net=host -v ~/.kube/config:/root/.kube/config:Z -v ~/cerberus.yaml:/root/cerberus/config/config.yaml:Z -d quay.io/openshift-scale/cerberus:latest")
```

Comment:
The kubeconfig path can be picked from the kraken config. This will use the default cerberus config, think we need to allow the user to tweak and mount the desired config into the container. We can add a cerberus config in kraken but we need to keep updating it in kraken whenever there's a new change in cerberus repo which is not really feasible. Maybe it's better to let the user take care of installing cerberus for now? Thoughts?